src/unstract/sdk/adapters/llm/no_op_llm/src/no_op_llm_adapter.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+from typing import Any
+
+from llama_index.core.llms import LLM
+
+from unstract.sdk.adapters.llm.llm_adapter import LLMAdapter
+from unstract.sdk.adapters.llm.no_op_llm.src.no_op_llm import NoOpLLM
+
+logger = logging.getLogger(__name__)
+
+
+class NoOpLLMAdapter(LLMAdapter):
+    def __init__(self, settings: dict[str, Any]):
+        super().__init__("NoOpLlm")
+        self.config = settings
+
+    @staticmethod
+    def get_id() -> str:
+        return "noOpLlm|f673a5a2-90f9-40f5-94c0-9fbc663b7553"
+
+    @staticmethod
+    def get_name() -> str:
+        return "No Op LLM"
+
+    @staticmethod
+    def get_description() -> str:
+        return "No Op LLM"
+
+    @staticmethod
+    def get_provider() -> str:
+        return "noOpLlm"
+
+    @staticmethod
+    def get_icon() -> str:
+        return "/icons/adapter-icons/noOpLlm.png"
+
+    @staticmethod
+    def get_json_schema() -> str:
+        f = open(f"{os.path.dirname(__file__)}/static/json_schema.json")
+        schema = f.read()
+        f.close()
+        return schema
+
+    def get_llm_instance(self) -> LLM:
+        llm: LLM = NoOpLLM(wait_time=self.config.get("wait_time"))
+        return llm
+
+    def test_connection(self) -> bool:
+        llm = self.get_llm_instance()
+        if not llm:
+            return False
+        response = llm.complete(
+            "The capital of Tamilnadu is ",
+            temperature=0.003,
+        )
+        response_lower_case: str = response.text.lower()
+        if response_lower_case:
```

Comment:
@harini-venkataraman why do we perform this check? Shouldn't we return `True` always?

Reply:
Addressed